index.js

```diff
@@ -1,351 +1,341 @@
+
+		//.replace(/\,\s{0,}\(/ig,",~::~( ")
+		//.replace(/\,/ig,",~::~"+tab+tab+"")
+
+		.replace(/ ALL /ig," ALL ")
+		.replace(/ AS /ig," AS ")
+		.replace(/ ASC /ig," ASC ")
+		.replace(/ DESC /ig," DESC ")
+		.replace(/ DISTINCT /ig," DISTINCT ")
+		.replace(/ EXISTS /ig," EXISTS ")
+		.replace(/ NOT /ig," NOT ")
+		.replace(/ NULL /ig," NULL ")
+		.replace(/ LIKE /ig," LIKE ")
+		.replace(/\s{0,}SELECT /ig,"SELECT ")
+		.replace(/\s{0,}UPDATE /ig,"UPDATE ")
+		.replace(/ SET /ig," SET ")
+
+		.replace(/~::~{1,}/g,"~::~")
+		.split('~::~');
+}
+
+vkbeautify.prototype.sql = function(input_text,step) {
+
+	var ar_by_quote = input_text.replace(/\s{1,}/g," ")
+							.replace(/\'/ig,"~::~\'")
+							.split('~::~'),
+		len = ar_by_quote.length,
+		ar = [],
+		deep = 0,
+		tab = this.step,//+this.step,
+		inComment = true,
+		inQuote = false,
+		parenthesisLevel = 0,
+		str = '',
+		ix = 0,
+		shift = step ? createShiftArr(step) : this.shift;;
+
+		for(ix=0;ix<len;ix++) {
+			if(ix%2) {
+				ar = ar.concat(ar_by_quote[ix]);
+			} else {
+				ar = ar.concat(split_sql(ar_by_quote[ix], tab) );
+			}
+		}
+
+		len = ar.length;
+		for(ix=0;ix<len;ix++) {
+
+			parenthesisLevel = isSubquery(ar[ix], parenthesisLevel);
+
+			if( /\s{0,}\s{0,}SELECT\s{0,}/.exec(ar[ix]))  {
+				ar[ix] = ar[ix].replace(/\,/g,",\n"+tab+tab+"")
+			}
+
+			if( /\s{0,}\s{0,}SET\s{0,}/.exec(ar[ix]))  {
+				ar[ix] = ar[ix].replace(/\,/g,",\n"+tab+tab+"")
+			}
+
+			if( /\s{0,}\(\s{0,}SELECT\s{0,}/.exec(ar[ix]))  {
+				deep++;
+				str += shift[deep]+ar[ix];
+			} else
+			if( /\'/.exec(ar[ix]) )  {
+				if(parenthesisLevel<1 && deep) {
+					deep--;
+				}
+				str += ar[ix];
+			}
+			else  {
+				str += shift[deep]+ar[ix];
+				if(parenthesisLevel<1 && deep) {
+					deep--;
+				}
+			}
+			var junk = 0;
+		}
+
+		str = str.replace(/^\n{1,}/,'').replace(/\n{1,}/g,"\n");
+		return str;
+}
+
+vkbeautify.prototype.xmlmin = function(input_text, preserveComments) {
+
+	var str = preserveComments ? input_text
+							   : input_text.replace(/\<![ \r\n\t]*(--([^\-]|[\r\n]|-[^\-])*--[ \r\n\t]*)\>/g,"")
+									 .replace(/[ \r\n\t]{1,}xmlns/g, ' xmlns');
+	return  str.replace(/>\s{0,}</g,"><");
+}
+
+vkbeautify.prototype.jsonmin = function(input_text) {
+
+	if (typeof JSON === 'undefined' ) return input_text;
+
+	return JSON.stringify(JSON.parse(input_text), null, 0);
+
+}
+
+vkbeautify.prototype.cssmin = function(input_text, preserveComments) {
+
+	var str = preserveComments ? input_text
+							   : input_text.replace(/\/\*([^*]|[\r\n]|(\*+([^*/]|[\r\n])))*\*+\//g,"") ;
+
+	return str.replace(/\s{1,}/g,' ')
+			  .replace(/\{\s{1,}/g,"{")
+			  .replace(/\}\s{1,}/g,"}")
+			  .replace(/\;\s{1,}/g,";")
+			  .replace(/\/\*\s{1,}/g,"/*")
+			  .replace(/\*\/\s{1,}/g,"*/");
+}
+
+vkbeautify.prototype.sqlmin = function(input_text) {
+	return input_text.replace(/\s{1,}/g," ").replace(/\s{1,}\(/,"(").replace(/\s{1,}\)/,")");
+}
+
+module.exports = new vkbeautify();
```

Comment:
Why this change as original library has `window.vkbeautify = new vkbeautify();`
Ref: https://github.com/vkiryukhin/vkBeautify/blob/master/vkbeautify.js